### ISSUE TITLE
WCAG 4.1.2 test 10 false positive; PDF/UA 7.1 test 9 possible false negative

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#76]
[This very small PDF document](https://github.com/veraPDF/veraPDF-wcag-algs/files/6970065/test-document-11.pdf) correctly raises error PDF/UA 7.4.2 test 1, as the document (deliberately) has a wrong ordering of headings. But the checker also appears to generate a false positive and a false negative:

- **WCAG 4.1.2 test 10 "Heading is incorrectly tagged as a paragraph"** points to a line that is obviously a paragraph and not a heading. This is a false positive, the checker should have recognised this line as a normal paragraph.
- In the Word source document I (deliberately) set no title. In the export to PDF I'd also expect the title of the document not to be set. I'd expect the checker to raise error **PDF/UA 7.1 test 9 "Metadata stream does not contain dc:title"**, but this does not happen. Is this a false negative? It is possible that the `dc:terms` element is present in the metadata, but is empty. If so, it should also generate the error.

**Comment by bdoubrov** [comment]
The document metadata contains dc:title, which is empty. For the moment veraPDF just checks if the title is present. We'll add an extra check that it is also non-empty.

The false positive with the heading is related to the fact that the headings use the font Calibri-Light, which has smaller font weight than the regular text font (Calibri). On the other hand, the font size of headings is bigger. We'll improve heuristics in header recognition here to fix this false positive as well.

**Comment by jzuidweg** [comment]
Ok, thanks for picking this issue up!

**Comment by jzuidweg** [comment]
This issue has been tested and is solved. It can be closed.
